### PR TITLE
allow an optional options object as either the second or third parameter

### PR DESCRIPTION
**Pull request by warbrett**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -21,7 +21,12 @@ function resetReducer(initialState, reducer){
   };
 }
 
-function promiseHandler(type, namespace){
+function promiseHandler(type, namespace, options){
```
Comment (monteslu) [inline index.js]: was thinking something more like `function promiseHandler(type, options){`

and 
```javascript
 if(typeof namespace === 'string') {
  options = {namespace: namespace};
}
```
and finally:

```javascript
var name = (options.namespace ? options.namespace + '-' : '') + type;
```

**Comment by monteslu** [comment]
requires 100% code coverage to pass :)

**Comment by warbrett** [comment]
Didn't even realize there were tests! One minute :)

**Comment by warbrett** [comment]
Updated!

**Pull request merged**